circonusllhist.go

```diff
@@ -456,6 +456,19 @@ func (h *Histogram) Mean() float64 {
 	return h.ApproxMean()
 }
 
+//Count returns the number of recorded values.
+func (h *Histogram) Count() uint64 {
+	if h.useLocks {
+		h.mutex.RLock()
+		defer h.mutex.RUnlock()
+	}
+	var count uint64
+	for _, bin := range h.bvs[0:h.used] {
+		count += bin.count
```

Comment:
In other languages we've done an early-out for NaN bins; not sure if that's needed with this approach.